cmd/milmove/serve.go

```diff
@@ -806,7 +806,16 @@ func serveFunction(cmd *cobra.Command, args []string) error {
 
 	// TRDM SOAP Request
 	trdmIsEnabled := v.GetBool(cli.TRDMIsEnabled)
-	if trdmIsEnabled {
+
+	// Do not execute LastTableUpdate in development environemnt
+	env := v.GetString(cli.EnvironmentFlag)
+	isDevEnv := env == "development"
+	if isDevEnv {
+		logger.Info(fmt.Sprintf("Skipping LastTableUpdate in the %s environemnt", env))
```

Comment:
"Environment" is typo'd

Reply:
Also, on line 822 could you replace `return err` with `logger.Fatal("error launching trdm last table update check", zap.Error(err))`